onnxruntime/core/providers/tensorrt/tensorrt_execution_provider_info.cc

```diff
@@ -87,7 +88,41 @@ ProviderOptions TensorrtExecutionProviderInfo::ToProviderOptions(const TensorrtE
       {tensorrt::provider_option_names::kDecryptionLibPath, MakeStringWithClassicLocale(info.engine_decryption_lib_path)},
       {tensorrt::provider_option_names::kForceSequentialEngineBuild, MakeStringWithClassicLocale(info.force_sequential_engine_build)},
   };
+  return options;
+}
+
+ProviderOptions TensorrtExecutionProviderInfo::ToProviderOptions(const OrtTensorRTProviderOptions& info) {
+
+  std::string kInt8CalibTable_ = "";
+  if (info.trt_int8_calibration_table_name != nullptr)
+    kInt8CalibTable_ = MakeStringWithClassicLocale(info.trt_int8_calibration_table_name);
```

Comment:
this name implies a const class data member (kName_)
https://google.github.io/styleguide/cppguide.html#Variable_Names
also, MakeString() is not necessary for a const char*, std::string::operator=(const char*) would work.
but this could be made a constant, e.g.:
```
auto empty_if_null = [](const char* s) { return s != nullptr ? std::string{s} : std::string{}; };
const std::string kInt8CalibTable = empty_if_null(info.trt_int8_calibration_table_name);
```

Reply:
Thanks for the advice. will use std::string::operator=(const char*) instead.